packages/polar/test/fixture-projects/project-with-scripts/polar.config.js

```diff
@@ -0,0 +1,22 @@
+const accounts = [{
+  name: 'CS',
+  address: 'secret14e979dswq7zchs69p9g7t4sx4sd33x90c5858a',
+  mnemonic: "slam club view virus chalk inherit bread caution hour vacant rain math"
+}]
+
+task('example2', 'example task', async (_ret) => 28)
+task('example', 'example task', async (__, { run }) => run('example2'))
```

Comment:
What's the use of these?

Reply:
this is for testing purposes, not any specific use, but user can use this as well